src/tests/key-store-search.cpp

```diff
@@ -82,23 +83,22 @@ TEST_F(rnp_tests, test_key_store_search)
     for (size_t i = 0; i < ARRAY_SIZE(testdata); i++) {
         pgp_key_id_t keyid = {};
         assert_true(rnp_hex_decode(testdata[i].keyid, keyid.data(), keyid.size()));
-        list seen_keys = NULL;
+        std::vector<pgp_key_t *> seen_keys;
```

Comment:
I think it's more convenient to use `std::set` here

Reply:
@rrrooommmaaa That would require to satisfy https://en.cppreference.com/w/cpp/named_req/Compare, and if I understand correctly, pgp_key_t will not. Given that it's just for tests I don't see the reason to spend time on implement that. And, actually, now it works the same way as it worked before. What do you think?

Reply:
@ni4 Aren't we comparing pointers?

Reply:
@rrrooommmaaa Oh, my fault, somehow mislooked (how?) pointerness, sorry. Fixed and force-pushed.